packages/app/src/pages/endpoints/[endpointId]/_/content/types/table/index.tsx

```diff
@@ -27,6 +33,11 @@ type Props = {
     React.Dispatch<React.SetStateAction<Record<string, Sort>>>
   ];
 };
+
+type Value = any;
+type Key = string;
+type ObjectData = Record<Key, Value>;
```

Comment:
What value, key and Object ?

Reply:
You might be able to resolve Data type and naming issues if you change the placement of the showing row data function from this component to the Table component.

Reply:
I will change the placement of the showing row data function from this component to the Table component in order to solve the issues.